apps/web/src/components/borrow/table/AvailableToBorrowTable.tsx

```diff
@@ -88,10 +102,11 @@ export const AvailableToBorrowTable: FunctionComponent = () => {
           size="small"
           data-cy={'borrowBtn ' + data.symbol}
           disabled={
-            convertUSDToAssetValue(
-              availableToBorrowUSD,
-              data.priceInEth,
-              data.usdPriceEth,
+            bigNumberToNumber(
```

Comment:
this is not very self explanatory (at-least not to me, probably to others)
It would be nice to avoid inlining this type of code in JSX, we should create a consts with self explanatory name or a comment

Reply:
Fixed 👍